examples/dynamics/oscillator.ipynb

```diff
@@ -154,4 +156,4 @@
    "metadata": {}
   }
  ]
-}
```

Comment:
> I put in the linebreak at the end because I don't like the little :no_entry_sign: that GitHub puts if you don't have it

It is less likely to look at the actual raw source of the notebooks than to look at Python files. So the little :no_entry_sign: might not be as much of a problem in these files.